static/styles/portico.css

```diff
@@ -1508,6 +1503,26 @@ input.new-organization-button {
     font-weight: 700;
 }
 
+.markdown h1:hover,
+.markdown h2:hover,
+.markdown h3:hover {
+    cursor: pointer;
+}
+
+.markdown h1:hover:after,
+.markdown h2:hover:after,
+.markdown h3:hover:after {
+    display: inline-block;
+    font: normal normal normal 16px/1 FontAwesome;
```

Comment:
I think we want the font to be represented as a percentage; right now they look great for some sizes (h3) but kinda small for larger ones (e.g. h1).

Reply:
@timabbott Any ideas for the percentage? The GitHub documentation all have their icons at a set font size (16px)

Reply:
Hmm, yeah, I think you're right; what GitHub does better for larger headings is centering the icon vertically.  See if you can make that work?

Reply:
@timabbott Done
![screenshot at nov 20 17-24-05](https://user-images.githubusercontent.com/15116870/33050034-a0bf2dd0-ce17-11e7-9322-ea82c44236d7.png)